### ISSUE TITLE
0.3.3 Windows auto-update when miner is running gets blocked by User Account Controll popup

### ISSUE THREAD

**Issue by ghost** [tari-project/universe#475]
**Describe the bug**
The video below captures the full flow. 
The auto-update is blocked by the following messages:
First the power shell is breifly opened then immediately dismissed:
![Screenshot 2024-09-13 at 1 00 27 PM](https://github.com/user-attachments/assets/f2baa1e8-e959-4538-ac4f-7114199a6e44)

Followed by:
![Screenshot 2024-09-13 at 12 42 44 PM](https://github.com/user-attachments/assets/b4a30047-34ca-4ad1-9b18-9ff336a17117)
Expanded:
![Screenshot 2024-09-13 at 12 59 26 PM](https://github.com/user-attachments/assets/ac6defac-06b2-47de-a47f-f2c85f2706a6)

Once the user clicks Yes and the application begins to sync, the Windows security checks:
![Screenshot 2024-09-13 at 1 02 41 PM](https://github.com/user-attachments/assets/7f9a8eed-5a14-44c2-96fb-88ce189cc14c)

![Screenshot 2024-09-13 at 1 03 11 PM](https://github.com/user-attachments/assets/aa1c6e90-f14d-46a4-9f3b-56aac2f432b7)

![image](https://github.com/user-attachments/assets/4078eeb8-71b0-4d5e-8ddc-eb7420f6e565)

**To Reproduce**
Miner must be running when a new version is made available. Auto-update must be attempted when miner is running. 
https://www.loom.com/share/c9cf078fefff4563b9c834e2c4ed2248?sid=e54b1b7f-4d93-4000-8ea3-b20db39fff4e

**Expected behavior**
Our goal is to have the miner not require any manual interaction when starting or updating. 

**Desktop (please complete the following information):**
 - OS & Version: 
Windows 11
AMD Ryzen 5 5600GT with Radeon graphics 3.60 Hz

and 
Windows VM

**Comment by stringhandler** [comment]
Let's investigate

**Comment by mmrrnn** [comment]
might be fixed along with #591

**Comment by uforiaio** [comment]
I will add it to my testing on the builds to see if it is closed. The UAC isn't an issue at this point from my testing, but with the flow of updates the windows firewall (WF) overrides is causing issues. On some machines there may be one WF override request, on others it may be two or three. 

When this occurs, the peer connections are already established. After accepting the WF prompt, I have seen from log parsing that the peer connections can become orphaned (#890) and **"may"** be causing some of the mining on ghost chains.

**Comment by brianp** [comment]
Closing as stale for now. If this issue persists please comment here or feel free to open another ticket.
